### PR TITLE
dx-1011 included paymentToken and generatePaymentToken in Payment Menu

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

File: _includes/payment-order-other-features.md [hunk 1]
```diff
@@ -1093,6 +1269,17 @@ The structure of a problem message will look like this:
 
 {% include merchant-authenticated-consumer.md %}
 
+## One-click Payments
```
Comment (asbjornu) [inline _includes/payment-order-other-features.md:1272]:
```suggestion
## One-Click Payments
```

File: _includes/payment-order-other-features.md [hunk 2]
```diff
@@ -1093,6 +1269,17 @@ The structure of a problem message will look like this:
...
+
+One-Click Payments should be used if you want to present your own Payment Menu 
+in all channels you want to support. This means that if you are both on the 
+web and in-app you will need to build your own payment menu to be able to use 
+the One-Click Payment functionality. You will need to ask the system to generate
+a `paymentToken` (with `generatePaymentToken` in a `Purchase` or `Verify`
+request) to be able to use this feature. You will also need to turn the 
+`Payment Order` into instrument mode (only show one instrument) 
+with the input parameter instrument.
```
Comment (asbjornu) [inline _includes/payment-order-other-features.md:1281]: The `instrument` word needs to link to the description of that feature once it gets added in DX-1016.

```suggestion
with the request parameter `instrument`.
```

File: _includes/payment-order-purchase.md [hunk 3]
```diff
@@ -106,6 +192,7 @@ Content-Type: application/json
...
 | {% icon check %} | └➔&nbsp;`description`             | `string`     | The description of the payment order.                                                                                                                                                                                                                                                                                                                                                                         |
+| {% icon check %} | └➔&nbsp;`generatePaymentToken`    | `bool`       | `true` or `false`. Set this to `true` if you want to create a paymentToken for future use as [One Click Payments][one-click-payments].                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```
Comment (asbjornu) [inline _includes/payment-order-purchase.md:195]:
```suggestion
| {% icon check %} | └➔&nbsp;`generatePaymentToken`    | `bool`       | `true` or `false`. Set this to `true` if you want to create a `paymentToken` to use in future [One Click Payments][one-click-payments].                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 6
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5735993&bid=18763558)

File: _includes/payment-order-other-features.md [hunk 4]
```diff
@@ -713,6 +708,187 @@ operation.
...
+|                  | └─➔&nbsp;`orderReference`         | `string(50)` | The order reference should reflect the order reference found in the merchant's systems.                                                                                                                                                                                                                                                                                                                       |
+|                  | └─➔&nbsp;`subsite`                | `String(40)` | The subsite field can be used to perform [split settlement][split-settlement] on the payment. The subsites must be resolved with Swedbank Pay [reconciliation][settlement-and-reconciliation] before being used.                                                                                                                                                                                              |
```
Comment (payex-dev) [inline _includes/payment-order-other-features.md:840]: Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5735993)

File: _includes/payment-order-purchase.md [hunk 5]
```diff
@@ -147,3 +234,31 @@ Content-Type: application/json
...
+| └➔&nbsp;`amount`         | `integer`    | {% include field-description-amount.md %}                                                                                                                                                                                 |
+| └➔&nbsp;`vatAmount`      | `integer`    | {% include field-description-vatamount.md %}                                                                                                                                                                              |
+| └➔&nbsp;`description`    | `string(40)` | {% include field-description-description.md documentation_section="checkout" %}                                                                                                                                                          |
+| └➔&nbsp;`userAgent`      | `string`     | The [user agent][user-agent] string of the consumer's browser.                                                                                                                                                            |
```
Comment (payex-dev) [inline _includes/payment-order-purchase.md:252]: Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5735993)

File: _includes/payment-order-other-features.md [hunk 6]
```diff
@@ -713,6 +708,187 @@ operation.
...
+|                  | └➔&nbsp;`payer`                   | `object`     | The `payer` object containing information about the payer relevant for the payment order.                                                                                                                                                                                                                                                                                                                     |
+|        ︎︎︎         | └─➔&nbsp;`consumerProfileRef`     | `string`     | The consumer profile reference as obtained through [initiating a consumer session][initiate-consumer-session].                                                                                                                                                                                                                                                                                                |
```
Comment (payex-dev) [inline _includes/payment-order-other-features.md:842]: Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5735993)

File: _includes/payment-order-other-features.md [hunk 7]
```diff
@@ -713,6 +708,187 @@ operation.
...
+| └➔&nbsp;`metadata`       | `string`     | The URI to the `payments` resource where information about all underlying payments can be retrieved.                                                                                                                      |
+| └➔&nbsp;`payments`       | `string`     | The URI to the `payments` resource where information about all underlying payments can be retrieved.                                                                                                                      |
+| └➔&nbsp;`currentPayment` | `string`     | The URI to the `currentPayment` resource where information about the current – and sole active – payment can be retrieved.                                                                                                |
+| └➔&nbsp;`operations`     | `array`      | The array of possible operations to perform, given the state of the payment order. [See Operations for details][operations].                                                                                              |
```
Comment (payex-dev) [inline _includes/payment-order-other-features.md:888]: Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5735993)

File: _includes/payment-order-purchase.md [hunk 8]
```diff
@@ -147,3 +234,31 @@ Content-Type: application/json
...
+| └➔&nbsp;`metadata`       | `string`     | The URI to the `payments` resource where information about all underlying payments can be retrieved.                                                                                                                      |
+| └➔&nbsp;`payments`       | `string`     | The URI to the `payments` resource where information about all underlying payments can be retrieved.                                                                                                                      |
+| └➔&nbsp;`currentPayment` | `string`     | The URI to the `currentPayment` resource where information about the current – and sole active – payment can be retrieved.                                                                                                |
+| └➔&nbsp;`operations`     | `array`      | The array of possible operations to perform, given the state of the payment order. [See Operations for details][operations].                                                                                              |
```
Comment (payex-dev) [inline _includes/payment-order-purchase.md:261]: Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5735993)

File: _includes/payment-order-other-features.md [hunk 9]
```diff
@@ -713,6 +708,187 @@ operation.
...
+| {% icon check %} | └─➔&nbsp;`amount`                 | `integer`    | {% include field-description-amount.md %}                                                                                                                                                                                                                                                                                                                                                                     |
+| {% icon check %} | └─➔&nbsp;`vatAmount`              | `integer`    | {% include field-description-vatamount.md %}                                                                                                                                                                                                                                                                                                                                                                  |
+
+
+{:.table .table-striped}
```
Comment (payex-dev) [inline _includes/payment-order-other-features.md:865]: Codacy found an issue: [[no-consecutive-blank-lines] Remove 1 line before node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5735993)

**Pull request merged**